shopify/usage-subscription-template/web/components/PlanCard.tsx

```diff
@@ -60,7 +62,7 @@ export default ({
                 description: `${
                   trialCalculations(
                     shop?.usedTrialMinutes,
-                    shop?.trialStartedAt,
+                    shop?.trialStartedAt as Date,
```

Comment:
Maybe we should modify the signature of `trialCalculations` instead of typecasting?